src/grift/insert-casts.rkt

```diff
@@ -147,7 +219,8 @@
     [(Gbox e) (Gbox (ic-expr e))]
     [(Gunbox (and (Ann _ (cons e-src e-ty)) (app ic-expr e)))
      (cond
-       [(GRef? e-ty) (Gunbox e)]
+       ;; Don't need to unfold-possible-mu here because cannon types
```

Comment:
cannon types?w

Reply:
This is a typo. It should be the word canon (as in canonical form). I will elaborate this comment.